tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "module": "ESNext",
```

Comment:
TS somewhat complains about this `Option 'module' must be set to 'NodeNext' when option 'moduleResolution' is set to 'NodeNext` ts 5.3.3

Reply:
We could do it, it changes resolution model from directory specification into file specification:
```ts
import {} from 'abc';
```
to:
```ts
import {} from 'abc.js';
```

Reply:
Ok, i've changed it to use nodenext directly. Try it on your build to verify it.